packages/react-core/src/utils/processDroppedItems.ts

```diff
@@ -46,12 +45,16 @@ export async function processDroppedEntries(
     }
   };
 
-  // Process all items
+  // Filter out and process files from the data transfer items
   await Promise.all(
-    Array.from(dataTransferItems)
-      .filter((dataTransferItem) => dataTransferItem.kind === 'file')
-      .map((dataTransferItem) => dataTransferItem.webkitGetAsEntry())
-      .filter((entry): entry is FileSystemEntry => entry !== null)
+    dataTransferItems
+      .reduce<FileSystemEntry[]>(
+        (acc, item) =>
```

Comment:
Super nit: Could destructure `item` but it's arguable whether that's more or less readable

Reply:
I'd destructure since we use reference `webkitGetAsEntry` multiple times

Reply:
Agreed. Updated to destructure